job/job.go

```diff
@@ -84,9 +121,30 @@ func CreateFromIP(ip net.IP) (Job, error) {
 	if err != nil {
 		return Job{}, err
 	}
+
+	// check if there is an active workflow for the device
```

Comment:
this comment doesn't really make sense on this line

Reply:
done